### PR TITLE
fix(eslint-plugin-template): handle 'innerHtml' input properly

### PR DESCRIPTION
**Pull request by rafaelss95**
**Current behavior**:

We can pass html content by using `innerHTML` and `innerHtml`. The later isn't currently ignored by the rule.

```html
<h3 [innerHtml]="dangerouslySetHTML"></h3>
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

**Expected behavior**:

```html
<h3 [innerHtml]="dangerouslySetHTML"></h3> // Pass
```

<hr>

I also replaced conditionals by selectors to make it more readable and shorter.

### CODE REVIEW

File: packages/eslint-plugin-template/tests/rules/accessibility-elements-content.test.ts [hunk 1]
```diff
@@ -21,7 +21,7 @@ ruleTester.run(RULE_NAME, rule, {
   valid: [
     '<h1>Heading Content!</h1>',
     '<h2><app-content></app-content></h2>',
-    '<h3 [innerHTML]="dangerouslySetHTML"></h3>',
+    '<h3 [innerHtml]="dangerouslySetHTML"></h3>',
```
Comment (rafaelss95) [inline packages/eslint-plugin-template/tests/rules/accessibility-elements-content.test.ts:24]: Btw, there's already a test case on line 28 that test the `innerHTML`: 

https://github.com/angular-eslint/angular-eslint/blob/3bc9307b7db838fdf935395ec45a03ca54b58cf9/packages/eslint-plugin-template/tests/rules/accessibility-elements-content.test.ts#L28

Comment (JamesHenry) [reply packages/eslint-plugin-template/tests/rules/accessibility-elements-content.test.ts:24]: 👍

**Review by JamesHenry — APPROVED** [review]
Looks good, thanks a lot!

**Comment by JamesHenry** [comment]
I need to report this flakiness to verdaccio, it's so frequent on the integrations test job, particular on master...

**Comment by JamesHenry** [comment]
Reported it to verdaccio: https://github.com/verdaccio/verdaccio/issues/2038

**Pull request merged**